Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:focal
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install wget gdebi-core xvfb supervisor ffmpeg x264 aria2 xserver-xorg-video-all mesa-utils tigervnc-standalone-server python3 python3-pip xdotool xrdp x11-apps x11-common x11-session-utils x11-xkb-utils x11-xserver-utils x11proto-core-dev x11proto-dev xserver-common xserver-xorg-core xserver-xorg-video-nvidia-455 libxv1 x11-xserver-utils && pip3 install websockets
+RUN aria2c -x 16 "https://github.com/cyberbotics/webots/releases/download/R2021a/webots_2021a_amd64.deb" && gdebi -n webots_2021a_amd64.deb && rm webots_2021a_amd64.deb
+EXPOSE 1234
```

Comment:
@dbscoach Just out of curiosity, is there any particular reason behind exposing these ports in particular?

Reply:
Yes, 1234 is the default port for x3d-live-streaming, that's a remnant from my first docker-compose-file. It wouldn't even actually get exposed if we didn't have network=none because the run-command doesn't map it anywhere. I'll remove it.

Reply:
Thanks @dbscoach I was literally just curious :slightly_frowning_face: